goldenmaster/Plugins/ApiGear/Source/ApiGearOLink/Private/unrealolink.cpp

```diff
@@ -94,6 +118,7 @@ void UUnrealOLink::Disconnect_Implementation()
 	{
 		m_node->unlinkRemote(objectName);
 	}
+	processMessages();
```

Comment:
Why calling _processMessages_ here directly? Theoretically it could be called from any thread.
We probably need to find a way to delay the closing of the socket until the messages are processed.

Reply:
true, thx! but instead of waiting I think it is better to make it thread safe and sent all queued messages immediately 🤔